pstress/pstress-run.conf

```diff
@@ -20,7 +20,7 @@ PSTRESS_BIN=${SCRIPT_PWD}/../src/pstress-ps
 # executed. Default options added.                                             #
 # Please modify the parameters as per needs before starting the runs.          #
 ################################################################################
-DYNAMIC_QUERY_PARAMETER="--tables 10 --records 100 --log-all-queries"
+DYNAMIC_QUERY_PARAMETER="--tables 10 --records 100 --log-all-queries --check-preload"
```

Comment:
Why not enable this option by default. There should not be a need to explicitly pass this option. If a user don't want to pre-check tables, we can have an option --no-check-preload